open-ce/utils.py

```diff
@@ -82,6 +82,30 @@ class Argument(Enum):
                                         default=DEFAULT_BUILD_TYPES,
                                         help='Comma delimited list of build types, such as "cpu" or "cuda".'))
 
+    DOCKER_BUILD = (lambda parser: parser.add_argument(
+                                        '--docker_build',
+                                        action='store_true',
+                                        help="Perform a build within a docker container. "
+                                             "NOTE: When the --docker_build flag is used, all arguments with paths "
+                                             "should be relative to the directory containing open-ce. Only files "
+                                             "within the open-ce directory and local_files will be visible at "
+                                             "build time."))
+
+    USE_LOCAL_SCRATCH = (lambda parser: parser.add_argument(
+                                        '--use_local_scratch',
```

Comment:
May be worth adding what's being pulled locally, i.e. the `.cache` and `conda-bld` directories...if it gets too wordy I wouldn't worry about it.